Projects/UOContent/Items/Containers/Fillable Containers/FillableContainers.cs

```diff
@@ -30,23 +17,29 @@ public override void AcquireContent()
 public partial class FillableLargeCrate : FillableContainer
 {
     [Constructible]
-    public FillableLargeCrate() : base(0xE3D) => Weight = 1.0;
```

Comment:
The weights are supposed to follow the non-fillable containers, cause fun fact, the weight is 255 for most containers, I dunno why.

Reply:
Per discussion, weights don't really matter because they are not movable. That is fine.